docs/docs/user-guide/roles.md

```diff
@@ -0,0 +1,64 @@
+
+!!! warning "Users vs Roles"
+    Outside of Mathesar, it’s not uncommon for people to say _user_ when referring to a PostgreSQL _role_. However, within the context of Mathesar, users and roles are different things! Our documentation maintains this distinction pedantically. When we say "user", we mean a _Mathesar_ user, and when we say "role", we mean a _PostgreSQL_ role.
+
+## Managing your PostgreSQL roles from within Mathesar {:#managing}
+
+To see the roles available on your server, navigate to the **Database Settings** tab within the page for one of your connected databases. From there, you'll be able to add roles, drop roles, and edit role inheritance.
+
+Role passwords are stored in Mathesar's [internal database](./databases.md#internal) and encrypted with your [SECRET_KEY](../administration/configuration.md#secret_key).
+
+Renaming roles and modifying role properties (e.g. `LOGIN` status) is not supported within Mathesar.
+
+## LOGIN vs non-LOGIN roles {:#login}
+
+In PostgreSQL every role has a boolean `LOGIN` property which is either _true_ or _false_. A `LOGIN` role may have an associated password and thus be used to connect to the server; while a non-`LOGIN` role may not. Non-`LOGIN` roles are often used as "group" roles to be granted to other roles.
+
+Mathesar lets you see the `LOGIN` status of each role, but does not allow you to change it. If you need to change a role's `LOGIN` status, you'll need to do so directly in PostgreSQL.
+
+
+## Roles vis-à-vis databases
+
+In PostgreSQL, roles live within the _server_ and thus are not necessarily specific to individual databases. One role can be configured to access many different databases; and one database can be configured for access via many different roles. Within the Mathesar interface however, the list of roles is presented _inside_ the page. This structure exists for the sake of convenience &mdash; and because PostgreSQL requires that clients connect to a specific database, even to query the roles present on a server. Nonetheless, it is important to understand that when you add/remove PostgreSQL roles (or edit their child roles), your changes will be visible for all databases which share the same server.
+
+## Ownership
+
+Every PostgreSQL object (e.g. a [table](./tables.md), [schema](./schemas.md), etc.) has one and only one role said to be its "owner". By default the owner is set to the role which created the object.
+
+The owner generally can do anything directly to the object, but not necessarily other objects contained within it. For example, a role might own a schema but not have access to certain tables within the schema.
+
+## Privileges
```

Comment:
Might we need to explain "permissions" vs. "privileges" somewhere?

Reply:
Sounds good. Can you implement this?